docs(k8s): add update notification warning (#2319)
scaleway/docs-content: containers/kubernetes/reference-content/version-support-policy.mdx

With Scaleway Kubernetes you can use the latest version of Kubernetes as soon as it is officially released and certified by the [Cloud Native Computing Foundation](https://www.cncf.io/). The versions are usually available in Kubernetes Kapsule and Kubernetes Kosmos within a few days after the release.

<Message type="important">

>   Scaleway is committed to provide updates on the products lifecycle as follows:

Author: ```suggestion
  Scaleway is committed to provide updates on the product lifecycle as follows:
```

  - Notification of feature deprecation will be announced in the [changelog](/changelog/).

>   - Information regarding a feature's End of Life (EOL) will be communicated through tickets and [scheduled maintenance notices](https://status.scaleway.com/).

Reviewer: Can we specify "support tickets in the console" so it's clearer?

Author: ```suggestion
  - Information regarding a feature's End of Life (EOL) will be communicated through support tickets in the console and [scheduled maintenance notices](https://status.scaleway.com/).
```

>   Upon reaching EOL, Scaleway will either remove the feature or initiate a migration (if applicable). While Scaleway endeavors to maintain service continuity, this cannot be assured.

Reviewer: ```suggestion
  Upon reaching EOL, Scaleway will either remove the feature or initiate a migration (if applicable). While Scaleway endeavors to maintain service continuity, this cannot be guaranteed.
```

>   It is the responsibility of the user to stay informed about both the deprecation (via the changelog) and the EOL (via tickets) of features. Users are also expected to engage with transition plans proactively to prevent any service disruption.

Reviewer: ```suggestion
  It is the responsibility of the user to keep themselves informed about both the deprecation (via the changelog) and the EOL (via tickets) of features. Users are also expected to engage with transition plans proactively to prevent any service disruption.
```
